### PR TITLE
Removes Node-Sass folder and dependency 

### PR DESCRIPTION
**Pull request by alizeleal**
### Description

The project no longer makes use of Node-Sass component, therefore it was removed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,10 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.4.5",
-    "@catho-private/catho-components": "^5.10.0",
+
```
Comment (MarcosViniciusPC) [inline package.json]:
```suggestion
```

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[] Code review

Task Runners
[] Build
[] Dev

Delete widget import in pages/render.js file

When I gave a yarn to install the dependencies, the yarn.lock file was updated and I want to know if this situation happened to you?Can you test this? The test is simple, delete the node_modules folder and reinstall the dependencies using the `yarn` command. If the yarn.lock file is updated, commit this

image

File: pages/cathocomponents.js [hunk 2]
```diff
@@ -29,9 +24,9 @@ export default function CathoComponentsPage() {
         gtmPrefix="next-header-b2c-nosearch"
         hasSearch
       />
-      <HeaderLoggedResponsive userProfile="STATUS_I_PROFISSIONAL" hasSearch />
```
Comment (rapahaeru) [inline pages/cathocomponents.js:32]: This component  `HeaderLoggedResponsive` still exists in catho components, how about making it come back with it?

**Review by rapahaeru — CHANGES_REQUESTED** [review]

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review

Task Runners
[x] Build
[x] Dev

**Review by hawkjr97 — APPROVED** [review]
## Review
- [x] Code review
- [x] Build
- [x] Dev

**Pull request merged**